website/source/docs/agent/configuration/telemetry.html.md

```diff
@@ -58,6 +58,20 @@ The following options are available on all telemetry configurations.
 - `publish_node_metrics` `(bool: false)` - Specifies if Nomad should publish
   runtime metrics of nodes.
 
+- `backwards_compatible_metrics` `(bool: false)` - Specifies if Nomad should
+  publish metrics that are backwards compatible with versions below 0.7, as
+  post version 0.7, Nomad emits tagged metrics. and all new metrics will
```

Comment:
". and all new".

Should be "tagged metrics. All new metrics will only be added to tagged metrics... ".